docs/root/intro/comparison.rst

```diff
@@ -35,11 +35,7 @@ features. Envoy provides the following main advantages over haproxy as a load ba
 
 * HTTP/2 support.
 * Pluggable architecture.
-* Multi-threaded architecture. It is substantially easier to operate and configure circuit breaking
-  settings when a single process is deployed per machine versus potentially multiple processes.
-* Integration with a remote service discovery service.
```

Comment:
Can you provide more details on this one? HAProxy does not include anything like the level of configuration Envoy does via remote xDS APIs, so I would prefer to rephrase this vs. delete. The rest of the change LGTM given recent 1.8 changes.

Reply:
Maybe I'm not clear on what level of configuration is offered by envoy. I'm guessing you don't consider it comparable to the haproxy admin socket?

Reply:
Definitely not. See here: https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/dynamic_configuration

If anything, the discrepancy has gotten more pronounced over time and this item needs to be made stronger. I would leave this line item, I'm very happy to merge the other changes.

Reply:
Re: the pluggable architecture, does Haproxy's lua support (since ~1.6) count as a viable alternative?

Reply:
Nope